servicebus/data-plane/azure-servicebus/src/main/java/com/microsoft/azure/servicebus/management/TopicDescription.java

```diff
@@ -132,16 +128,16 @@ public Duration getAutoDeleteOnIdle() {
      * The minimum duration is 5 minutes.
      */
     public void setAutoDeleteOnIdle(Duration autoDeleteOnIdle) {
-        if (autoDeleteOnIdle != null &&
-                autoDeleteOnIdle.compareTo(ManagementClientConstants.MIN_ALLOWED_AUTODELETE_DURATION) < 0)
-        {
+        if (autoDeleteOnIdle != null
+            && autoDeleteOnIdle.compareTo(ManagementClientConstants.MIN_ALLOWED_AUTODELETE_DURATION) < 0) {
             throw new IllegalArgumentException(
                     String.format("The value must be greater than %s.",
                             ManagementClientConstants.MIN_ALLOWED_AUTODELETE_DURATION));
         }
 
         this.autoDeleteOnIdle = autoDeleteOnIdle;
-        if (this.autoDeleteOnIdle.compareTo(ManagementClientConstants.MAX_DURATION) > 0) {
+        if (this.autoDeleteOnIdle != null
```

Comment:
[SpotBugs-P2]: Possible null pointer dereference of autoDeleteOnIdle in com.microsoft.azure.servicebus.management.TopicDescription.setAutoDeleteOnIdle(Duration)